src/cockpit/packages.py

```diff
@@ -195,6 +225,17 @@ def get_content_security_policy(self, origin):
     def serve_file(self, path, channel):
         filename = self.negotiate_file(path, channel.headers)
 
+        # HACK: if a translation file is missing, just return empty
+        # content. This saves a whole lot of 404s in the developer
+        # console when trying to fetch po.js for English, for example.
+        if filename is None and path.endswith('po.js'):
+            headers = {
+                "Access-Control-Allow-Origin": channel.origin,
+            }
+            channel.http_ok("application/javascript", headers)
+            channel.send_data(b"")
```

Comment:
Is that necessary?

Reply:
This needs to be discussed, it seemed irrelevant as a default make generates translations. So maybe this is for cockpit-podman? I rather retain the behaviour and later rip it out.

Reply:
I meant only the one line — sending the empty data.  I think that's a no-op and can be excluded — but see above.